google/cloud/storage/internal/grpc_client.cc

```diff
@@ -48,60 +59,54 @@ bool DirectPathEnabled() {
                         [](absl::string_view v) { return v == "storage"; });
 }
 
-std::string GrpcEndpoint() {
+Options FillWithDefaultsGrpc(Options options) {
+  options = FillWithDefaults(std::make_shared<oauth2::AnonymousCredentials>(),
```

Comment:
I assume you saw the naming pattern used by Spanner (https://github.com/googleapis/google-cloud-cpp/blob/master/google/cloud/spanner/internal/defaults.h) and the generator, but following that pattern, this would be named `DefaultOptions()`, with a ` = {}` default argument. I'm  happy to go with whatever name you choose, but just pointing out the difference in case you didn't see that.

Reply:
Done.  I was not paying attention to be candid.